client/src/main/java/org/spine3/base/RegistryKey.java

```diff
@@ -21,8 +21,8 @@
 package org.spine3.base;
 
 /**
- * A marker interface, serves as the type identifier
- * for the keys into {@code StringRegistry} warehouse.
+ * A marker interface, serves as the type identifier for the keys
+ * of the {@code StringRegistry} warehouse to save and retrieve the {@code Stringifier} instances.
```

Comment:
In IT `warehouse` usually means something much bigger than `StringRegistry`. Please find another word.